packages/block-library/src/navigation-submenu/index.php

```diff
@@ -249,14 +183,33 @@ function render_block_core_navigation_submenu( $attributes, $content, $block ) {
 	}
 
 	if ( $has_submenu ) {
-		$colors      = block_core_navigation_submenu_build_css_colors( $block->context, $attributes, $has_submenu );
-		$classes     = array_merge(
-			array( 'wp-block-navigation__submenu-container' ),
-			$colors['css_classes']
-		);
-		$css_classes = trim( implode( ' ', $classes ) );
+		// Copy some attributes from the parent block to this one.
+		// Ideally this would happen in the client when the block is created.
```

Comment:
Why is it ideal?

Reply:
I was thinking that then the sub-menu block would be the one to control it, rather than being orchestrated by the parent.

Reply:
I'm not sure. The reason it's orchestrated by the parent is so you can set the colors for _all_ children in a single place rather than have to visit every single child block and configure the colors.

We could make it so changing one child block's colors changes all of them but I think that's very confusing. 

I would consider leaving the controls in place on the Nav block but allowing the values set their to determine the _defaults_ for controls we expose on the child blocks. 

That means we still have a single place to se the colors.

Let me know if I've misunderstood here.

Reply:
Absolutely.